### ISSUE TITLE
index.js contains DOS line endings since version 5.0.1

### ISSUE THREAD

**Issue by sluukkonen** [gilamran/tsc-watch#149]
I noticed a small issue with the latest releases: if I install tsc-watch using yarn and try to use it on a macOS machine, I get the following error:

```
env: node\r: No such file or directory
```

This is caused by index.js containing DOS line endings in 5.0.1 and 5.0.2. Version 4.6.2 works without issues.

```
❯ xxd node_modules/.bin/tsc-watch # 5.0.2
00000000: 2321 2f75 7372 2f62 696e 2f65 6e76 206e  #!/usr/bin/env n
00000010: 6f64 650d 0a0d 0a72 6571 7569 7265 2827  ode....require('
00000020: 2e2f 6c69 622f 7473 632d 7761 7463 6827  ./lib/tsc-watch'
00000030: 293b 0d0a                                );..
❯ xxd node_modules/.bin/tsc-watch # 4.6.2
00000000: 2321 2f75 7372 2f62 696e 2f65 6e76 206e  #!/usr/bin/env n
00000010: 6f64 650a 0a72 6571 7569 7265 2827 2e2f  ode..require('./
00000020: 6c69 622f 7473 632d 7761 7463 6827 293b  lib/tsc-watch');
00000030: 0a
```

Perhaps the 5.0.x versions were published on a Windows machine with the git repository set to use DOS line endings?

**Comment by cubismod** [comment]
Running into this on my Linux system as well.

**Comment by gilamran** [comment]
fixed (hopefully) on v5.0.3
thanks for raising the flag!
